docs/changelog/95329.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 95329
+summary: Change default of `ignore_malformed` to `true` in `logs-*-*` data streams
```

Comment:
@felixbarny I would like us to have somewhere a longer explanation around this change for our users. So someone upgrading and seeing more data gets accepted understands what happened. Should we just make the changelog slightly longer or have it somewhere in the docs?

Reply:
I guess it doesn't hurt to add a bit more context here. But we'll probably want to have a similar note for integrations.

Maybe something like:
This is our first step in a journey to make the default mappings for logs more lenient.

Trying to index the wrong data type into a field throws an exception by default, and rejects the whole document. The <<ignore-malformed,`ignore_malformed`>> parameter, if set to `true`, allows the exception to be ignored.

Reply:
The eventual [release notes](https://www.elastic.co/guide/en/elasticsearch/reference/current/release-notes-8.7.0.html) do not contain much room for elaborated explanations on each changelog entry.
I will add this description to the PR description instead and we'll look for a better way to make these changes visible to users.